internal/rollup/rollup_bundle.bzl

```diff
@@ -664,13 +690,22 @@ ROLLUP_ATTRS = {
 
 ROLLUP_OUTPUTS = {
     "build_cjs": "%{name}.cjs.js",
+    "build_cjs_map": "%{name}.cjs.js.map",
```

Comment:
this needs some re-organization as a starlark provider so the JS and .map can stay together as tuple through the dependent build steps...

Reply:
This seems like the ideal use case for OutputGroupInfo provider
https://docs.bazel.build/versions/master/skylark/rules.html#requesting-output-files
https://docs.bazel.build/versions/master/skylark/lib/OutputGroupInfo.html

Reply:
:+1:

Would we still want the targets for individual files? Only the `.js` ones which exist today?

Reply:
I got this working locally, but does this mean people must use the `--output_groups` flag instead of something similar to the `bundle.es2015.js` used today?